cleanlab_studio/internal/enrichment_utils.py

```diff
@@ -52,9 +52,9 @@ def get_regex_match_or_replacement(
     # if the regex is a string, do matching
     if isinstance(regex, str):
         compiled_pattern = re.compile(regex)
-        pattern_match = compiled_pattern.match(response)
-        if pattern_match:
-            return pattern_match.group(1)
+        pattern_match = compiled_pattern.findall(response)
+        if len(pattern_match) > 0:
+            return str(pattern_match[0])
         return None
```

Comment:
previously, there was discussion if this is best returned as None or the initial str. This would be a good PR to change that if necessary.